tests/cases/tm2tm/scripts/test-service

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+
+: <<'END_COMMENT'
+* relay-interval = 10s
+* src-relay-optimize-interval = 15s
+* src-relay-optimize-count = 3
+* dst-relay-optimize-interval = 15s
+* dst-relay-optimize-count = 3
+
+- relay service [query = 0, time = 0] -> skip
+    - transfer
+    - 
+    - 
+    - finality
+    - query = 1
+    - 
+    - 
+    - 
+    - 
+- relay service [query = 1, time = 0] -> skip
+    - transfer
+    - 
+    - 
+    - finality
+    - query = 2
+    - 
+    - 
+    - 
+    - 
+- relay service [query = 2, time = 10] -> skip
+    - 
+    - 
+    - 
+    - 
+    - 
+    - 
+    - 
+    - 
+    - 
+- relay service [query = 2, time = 20] -> exec (time)
+    - transfer x 3
+    - 
+    - 
+    - finality
+    - query 3
+    - 
+    - 
+    - 
+    - 
+- relay service [query = 3, time = 10] -> exec (count)
+END_COMMENT
+
+set -eux
+
+SCRIPT_DIR=$(cd $(dirname $0); pwd)
+RLY_BINARY=${SCRIPT_DIR}/../../../../build/yrly
+RLY="${RLY_BINARY} --debug"
+
+TM_ADDRESS0=$(${RLY} tendermint keys show ibc0 testkey)
+TM_ADDRESS1=$(${RLY} tendermint keys show ibc1 testkey)
+
+RETRY_COUNT=10
+RETRY_INTERVAL=1
+
+${RLY} service start ibc01 --relay-interval 10s --src-relay-optimize-interval 15s --src-relay-optimize-count 3 --dst-relay-optimize-interval 15s --dst-relay-optimize-count 3 &
+RLY_PID=$!
+
+# transfer a token
+${RLY} tx transfer ibc01 ibc0 ibc1 100samoleans ${TM_ADDRESS1}
```

Comment:
Are `TM_ADDRESS0` and `TM_ADDRESS1` used by both the relayer and the transferer?